### DISCUSSION TITLE
is "enable :create_account": always implicit?

### DISCUSSION THREAD

**Discussion by jf** [jeremyevans/rodauth#406] [General]
I was just wondering why some examples do not have `:create_account` enabled. e.g. at http://rodauth.jeremyevans.net/rdoc/files/doc/guides/delay_password_rdoc.html:

```
plugin :rodauth do
  enable :login, :logout, :verify_account
  verify_account_set_password? true
end
```

I was wondering how these "flags" (or "enable"s) would make sense, if `:create_account` isn't there. Is this meant for when accounts are pre-populated? But upon testing this particular example, it seems that `:create_account` is there; but perhaps just implicitly enabled? Because if I were to go to a protected route, the login form that's given to me shows with a "Create a New Account" link that works.

On that note... is `:create_account` always implicitly enabled? or is there some situation where if `:create_account` isn't there in the list, that it's actually *not* enabled?

**Comment by jeremyevans** [comment]
In this particular example, `verify_account` depends on `create_account`, so you don't need to separately enable `create_account`.

As a general rule, the example configurations given in the guides are designed to demonstrate the specific issue being discussed (e.g. delaying password setting until verification), so there may be cases where the configuration appears inconsistent (e.g. `logout` but no `login`).

**Reply by jf** [reply]
ok. I will take note of that.
